src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java

```diff
@@ -365,17 +365,22 @@ void consumeRecords(final List<O> recOps) {
       val consumerConfig = createConsumerConfig(nodeAddr);
       // set the records count limit equal to the remaining read operations count
       consumerConfig.put(ConsumerConfig.MAX_POLL_RECORDS_CONFIG, remainingOpCount);
+      var recOp = (DataOperation) null;
       try (val consumer = new KafkaConsumer(consumerConfig)) {
         try {
           concurrencyThrottle.acquire();
-          var recOp = (DataOperation) null;
           val topics = new HashSet<>();
           // mark the request time for all remaining operations
           for (var i = opCount - remainingOpCount; i < opCount; i++) {
             recOp = (DataOperation) recOps.get(i);
             recOp.startRequest();
             recOp.finishRequest();
-            topics.add(recOp.dstPath());
+            val topicName =
+                recOp
+                    .srcPath()
+                    .substring(
+                        recOp.srcPath().contains("/") ? recOp.srcPath().indexOf("/") + 1 : 0);
```

Comment:
Isn't it easier to use the method .replace("/","")? In my opinion, it can be more obvious.